types/artifacts.d.ts

```diff
@@ -409,10 +409,23 @@ declare global {
         attributeWidth: string;
         /** The raw height attribute of the image element. CSS images will be set to the empty string. */
         attributeHeight: string;
-        /** The CSS width property of the image element. */
+        /**
+         * The CSS width property of the image element.
+         * @deprecated Use `element?.cssSizing.width` instead.
+         */
         cssWidth?: string;
-        /** The CSS height property of the image element. */
+        /**
+         * The CSS height property of the image element.
+         * @deprecated Use `element?.cssSizing.width` instead
+         */
         cssHeight?: string;
+        /** The width/height of the element as defined by matching CSS rules. Set to `undefined` if the data was not collected. */
+        cssSizing: undefined | {
```

Comment:
`undefined` will have the property be dropped when saved to JSON, so this is de facto 
```suggestion
        cssSizing?: {
```

Reply:
thx. good call.